docs/topics/collections-overview.md

```diff
@@ -287,3 +293,28 @@ The default implementation of `MutableMap` – [`LinkedHashMap`](https://kotlinl
 preserves the order of elements insertion when iterating the map.
 In turn, an alternative implementation – [`HashMap`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/-hash-map/index.html) – 
 says nothing about the elements order.
+
+### ArrayDeque
+
+[`ArrayDeque<T`>](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/-array-deque/) is an implementation of a double-ended queue, which allows you to add or remove elements both at the beginning or end of the queue.
```

Comment:
For purposes of SEO, it might be worth mentioning here that as such, `ArrayDeque` also fills the role of both a Stack and Queue data structure in Kotlin. (keyword here being `Stack` – it's one of the basic data structures we don't have in Kotlin, but that can be conveniently emulated using an ArrayDeque)